openpgp/read_test.go

```diff
@@ -819,7 +819,7 @@ func TestCorruptedMessageInvalidSigHeader(t *testing.T) {
 	promptFunc := func(keys []Key, symmetric bool) ([]byte, error) {
 		return passphrase, nil
 	}
-	const expectedErr string = "openpgp: invalid data: parsing error"
+	const expectedErr string = "openpgp: decryption with session key failed: parsing error"
```

Comment:
Let's add similar tests for SEIPDv2 to test (and "demo") what kind of errors are thrown with AEAD (hopefully the AEAD chunks are not released/parsed even with streaming).

Reply:
Added a test for a missing last aead authentication tag:
https://github.com/ProtonMail/go-crypto/pull/237/commits/6525f786bfab3cfde3c6d5fbfc70e615a8f79de8